### ISSUE TITLE
Tesseract jumped from 22MB to 682MB because it now includes all language data

### ISSUE THREAD

**Issue by varenc** [Homebrew/homebrew-core#36760]
- [x] are reporting a bug others will be able to reproduce and not asking a question or requesting software. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh. To get software added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)
- [ ] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, Homebrew/homebrew-core formula (not cask) on macOS? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a Linux problem please file this issue at https://github.com/Linuxbrew/homebrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?  (_not relevant_)

#### To help us debug your issue please explain
The Tesseract formula's on disk size jumped from 22MB to 682MB after #36293 which removed all options.  Specifically, [this change](https://github.com/Homebrew/homebrew-core/commit/012aea3513f0f548ac98267204166bf7cf0dd0e1#diff-00cd9f64b1f8e926676b8684978fa52cR59) now builds tesseract with support for 125 languages and 37 scripts.  Previously tesseract only included English and OSD (orientation detection) by default which added 14MB of tessdata.

####  What you expected to happen
I don't think the default tesseract should be 680MB and support 162 languages/scripts.  I think tesseract should just support english/osd out of the box like before and users that want additional language support can use a 3rd party tap to access that or other solutions the community provides.

####  Step-by-step reproduction instructions (by running `brew install` commands)
```
➜  brew uninstall tesseract
➜  brew install tesseract
➜  du -d 0 -h /usr/local/Cellar/tesseract/4.0.0
 682M	/usr/local/Cellar/tesseract/4.0.0
➜  brew uninstall tesseract
➜  brew install https://gist.github.com/varenc/314e491dc9ac75cdbce3d45e1a8ea0b5/raw/1c3b6b92bacab2d485845f2f58425acf5e89a72f/tesseract.rb  # <--- this is a copy of the formula before the recent change but with cxx11 removed
➜  du -d 0 -h /usr/local/Cellar/tesseract/4.0.0
 22M	/usr/local/Cellar/tesseract/4.0.0
```

You can also more directly just check the differences in bottle sizes between revision 1 and revision 2 like this:
```
➜ curl -sI https://homebrew.bintray.com/bottles/tesseract-4.0.0.mojave.bottle.1.tar.gz  | grep -i content-length | awk '{printf "%.2f MB\n", $2/1024**2}'
8.95 MB
➜ curl -sI https://homebrew.bintray.com/bottles/tesseract-4.0.0.mojave.bottle.2.tar.gz  | grep -i content-length | awk '{printf "%.2f MB\n", $2/1024**2}'
339.90 MB
```
(The bottle size is about half the on-disk size since training data seem to compress well)

####  My proposal
I'm happy to open a PR request that changes tesseract back to only including english and OSD. Alternatively maybe including the 10 most popular languages would be a reasonable tradeoff. I could also look into creating a seperate Formula that installs all of the tesseract language data. Though it would need to update the `TESSDATA_PREFIX` ENV to point to the new tessdata directory and a little hazy of if that's possible or if there's other solutions that could work.

**Comment by varenc** [comment]
Taking the thumbs up as encouragement, I made a PR that addresses this: #36786

**Comment by fxcoudert** [comment]
This was fixed in #36786. The `tesseract` formula now only includes basic language support, while additional support is available in the larger `tesseract-lang` formula.
